composables/useVerifyAccount.ts

```diff
@@ -30,36 +33,50 @@ export default function useVerifyAccount() {
     return signedMessage
   }
 
-  const getSignedMessage = async () => {
+  const getSignedMessage = async (message) => {
```

Comment:
```suggestion
  const getSignedMessage = async (message: string) => {
```
plus type the return